guides/source/configuring.md

```diff
@@ -481,6 +481,13 @@ in controllers and views. This defaults to `false`.
 
 * `config.active_record.enumerate_columns_in_select_statements` when true, will always include column names in `SELECT` statements, and avoid wildcard `SELECT * FROM ...` queries. This avoids prepared statement cache errors when adding columns to a PostgreSQL database for example. Defaults to `false`.
 
+* `config.active_record.destroy_all_in_batches` ensures
```

Comment:
Not related to this PR. Moved this to the right section, it was under the Schema Dumper config which was incorrect.